CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+# 13.0.0
+
+#### Breaking changes
+
+- In variables:
+  - Remove SimpleVariable and DatedVariable. Any Variable acts like old DatedVariable
+  - Remove start_date attribute
+  - Rename stop_date attribute to end
```

Comment:
When you are quoting code (for instance `stop_date`), it's better to use backticks (`) to format it properly.